hack/generate-cr.sh

```diff
@@ -5,10 +5,23 @@ export NS=${NS:-"dex-operator"}
 export APPS=$(oc get infrastructure cluster -ojsonpath='{.status.apiServerURL}' | cut -d':' -f2 | sed 's/\/\/api/apps/g')
 export GITHUB_APP_CLIENTID=${GITHUB_APP_CLIENTID:-"githubappclientid"}
 export GITHUB_APP_CLIENTSECRET=${GITHUB_APP_CLIENTSECRET:-"githubappclientsecret"}
+
 export CLIENT_NAME=${CLIENT_NAME:-"thing"}
 export CLIENT_SECRET=${CLIENT_SECRET:-"thing123456"}
 export CLIENT_SECRET_NAME=cluster1-clientsecret
 
+export DEXSERVER_CLIENT_NAME=dexserverclientname
+export DEXSERVER_CLIENT_ID=dexserverclientid
+export DEXSERVER_CLIENT_SECRET_NAME=dexserversecret-secret
+export DEXSERVER_CLIENT_SECRET=dexserversecret123456
```

Comment:
@cdoan1 Not seeing usages for these DEXSERVER_CLIENT_NAME, DEXSERVER_CLIENT_ID, DEXSERVER_CLIENT_SECRET_NAME, DEXSERVER_CLIENT_SECRET. Are they being used?

Reply:
yes, updated my script to use this now